README.md

```diff
@@ -5,14 +5,26 @@
 Before starting the server, copy .env.example as .env and fill in the TODO fields
 
 If you want to seed some test data run:
-```
-npm run seed
-```
 
-## API
+    $ npm run seed
+
+To startup the application, run
+
+    $ npm run watch-server
+
+It's possible to startup the application stack with docker-compose:
 
-### API definition
+    $ docker-compuse up -d # Starts the three services postgres, influx and api
 
+### Production
+To use the docker-compose stack in production, run the following:
+
+    $ cd <data-dir-with-dotenv>
+    $ docker image pull ase2019red/thingy-api-red
+    $ docker-compose -f <path-to-compose-file> up -d
```

Comment:
I think this can be adapted to stack deployments. So you are running Docker Swarm on a single node server? This does not really make sense, does it? Do you have a real benefit from the Swarm thing? For the sake of simplicity, I would suggest to use docker-compose otherwise.